docs/release-notes/7.16.asciidoc

```diff
@@ -0,0 +1,54 @@
+* Allows migration of detection rule actions to centralized Kibana alerting framework. Users may receive notifications sooner after alerts have been generated, depending on rule configuration and actions frequency ({pull}115243[#115243], {pull}115101[#115101]).
+* Changes the prebuilt indicator match rule's interval and lookback time to one hour ({pull}115185[#115185]).
+* Allows exceptions to be exported with rules ({pull}115144[#115144]).
+* Improves the formatting of array values and JSON in the *Table* and *JSON* tabs ({pull}115141[#115141]).
+* Providing users with a new, simpler way to add data to their environments through the {agent} ({pull}115016[#115016], {pull}112142[#112142]).
+* Enables the Index connector and action for the Detection engine ({pull}111813[#111813]).
+* Hides building block rules on the Overview page ({pull}105611[#105611]).
+* Stops the status badge from stretching in the Alert Details flyout ({pull}116237[#116237]).
+* Improves rule statuses if the user doesn't have permissions to the source index ({pull}115114[#115114]).
+* Fixes the exceptions export route ({pull}114920[#114920]).
+* Restores local storage persistence for the Alerts table and the Remove Column action ({pull}114742[#114742]).
+* Fixes issues that occurred when adding the {es-sec-endpoint} integration to an {agent} policy in {fleet} ({pull}114467[#114467]).
+* Updates the Indexing Time and Query Time columns in the Rule Monitoring table to be SUM, instead of MAX ({pull}114023[#114023]).
+* Fixes a bug that prevented modals on the Overview page from opening when the user clicked on the *Inspect* button ({pull}113161[#113161]).
+* Sets a new default indicator index query that checks the indicator index patterns for matched indicators that have occurred in the past 30 days ({pull}112300[#112300]).
+* Decodes file names on uploaded value lists and fixes a bug that stopped value lists from being deleted ({pull}111838[#111838]).
+* Fixes a bug that validated the form to create a trusted application when the name field was empty ({pull}111508[#111508]).
+
+[discrete]
+[[upcoming-breaking-changes-7.16.0]]
+==== Upcoming breaking changes
+To improve the detection engine's rule preview functionality in 8.0.0, preview alerts will be written to a new index called the signals preview index (`.siem-signals-preview*`). In order to view this index and use the updated rule preview functionality, roles must have `read` privileges to the new signals preview index. Also note that, other than their index lifecycle management policies, signal preview indices are nearly identical to existing signal indices.
+
+To give a role `read` privileges to the new signals preview index:
+
+. Open the main menu, then go to *Management -> Stack Management -> Security -> Roles*.
+. Select the custom role you want to update.
+. Modify the role's index privileges as follows:
+.. *Indices*: Enter the signals preview indices appropriate for the role and space, or spaces.
```

Comment:
End of sentence is a little confusing. Could it just say "for the role and space"?

Reply:
Are you recommending that "appropriate" be removed? That'd work, but I do want to keep the last bit about multiple spaces. When setting a role’s index privs, users can assign privs to a single space, multiple ones, or all spaces. I don’t think we use the parentheses “s” convention (e.g., space(s)) in our docs and it’s probably not advisable anyway since we’re writing to diverse audiences.

Reply:
I was confused over the last part about spaces. Now that I look at it again, it's not clear how spaces are involved when assigning index privileges to a role. There's a separate UI for assigning Kibana space privileges to a role, but that didn't seem part of this procedure involving index privileges.

Reply:
Replacing with the following:
Enter the signals preview indices that correspond with the signals indices. For example, the `.siem-signals-preview*` index pattern corresponds with the `.siem-signals*` index pattern. Similarly, the `.siem-signals-preview-<KIBANA-SPACE>*` index pattern corresponds with the `.siem-signals-<KIBANA-SPACE>*` index pattern.